src/GitHub.VisualStudio/UI/Views/PullRequestCreationView.xaml.cs

```diff
@@ -17,7 +17,8 @@ public partial class PullRequestCreationView : GenericPullRequestCreationView
         public PullRequestCreationView()
         {
             InitializeComponent();
-            DataContextChanged += (s, e) => ViewModel = e.NewValue as IPullRequestCreationViewModel;
+            // DataContextChanged += (s, e) => ViewModel = e.NewValue as IPullRequestCreationViewModel;
+            DataContextChanged += (s, e) => ViewModel = new GitHub.SampleData.PullRequestCreationViewModelDesigner() as IPullRequestCreationViewModel;
```

Comment:
I reverted this in edb905e, but can you double check to make sure this looks good?